docs/production/export-and-import.md

```diff
@@ -446,6 +444,27 @@ rm -rf /home/zulip/uploads/*/2/
 
 Once that's done, you can simply re-run the import process.
 
+## Compliance exports
+
+In some circumstances, corporate or legal compliance may require performing
+selective data exports for messages containing certain keywords, from particular
+time-frames, or from particular users.
+
+This can be done with the `export_search` command-line tool. For example, to
+search for messages containing the word "server" between November 1st and 6th,
+from `alice@example.com`:
+
```

Comment:
Maybe something like this to make it easier to parse?


In some circumstances, corporate or legal compliance may require performing
selective data exports. Zulip let you specify the following parameters when exporting messages:

- Keywords in the message text
- Message sender
- Time range for when messages were sent

Reply:
Instead?  I don't know how much to just tell people to try `./manage export_search --help` to find out the names of the options, vs hardcode them here in the doc.

Reply:
I think explicitly telling people to run `./manage export_search --help` would be a good idea. That's something that won't necessarily be obvious to many people this particular tool is aimed at.

Also, some text like Alya suggested may be helpful even without giving the exact names/syntax of the options. The person reading this may not be the person who can log into the server to run the command, but someone who wants to know what sorts of things they can ask that latter person to do with it.

Reply:
My proposal was to replace lines 453-455, not touching the rest of the instructions.

Reply:
Oh, I think I may have left my comment a bit too low in the file? Anyway, my proposal was to replace the first part (lines 449-451), leaving the rest untouched.

Reply:
I applied a version of this, avoiding the "Zulip lets you" in favor of specifying "This tool ...".